### PR TITLE
Fix #1514: variant_type must be remembered.

### LINKED ISSUE #1514: research page 2 is a clinical page
Hi , see the exemple of fam 83
If we are on the research page and click on "next" the fololowing page is a clinical page.

initial page
https://scout.scilifelab.se/cust003/83/variants?variant_type=research
click on "NEXT"
https://scout.scilifelab.se/cust003/83/variants
this come up: on top it states clinical SNV and INDEL

thanksfor help

### PR DESCRIPTION
**Pull request by dnil**
This PR fixes a bug.

**How to test**:
1. on stage or local with research loaded (click request research in a demo instance, then run ```scout --demo load research```
1. with the current master, note that when using next page, first page or reset filters on a research page, variant_type resets to the default clinical.
1. with this PR, that is not supposed to happen: research stays research, clinical stays clinical. This should hold for both SNVs and SVs.

**Expected outcome**:
The functionality should be working

**Review:**
- [x] code approved by CR
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Is this ready for review?

**Comment by dnil** [comment]
Hm, I guess - a little complicated to make (simple) automated tests for: suggestions are welcome.

**Comment by northwestwitch** [comment]
Not related to the bug, but as a result of the bootstrap 4, these the X to close the research modal looks ugly: 
image

Same for the rerun one. This could be fix by using this code
```
<h4 class="modal-title">Request research list</h4>
   <button type="button" class="close" data-dismiss="modal" aria-label="Close">
    <span aria-hidden="true">&times;</span>
</button>
```

instead of:
```
<button type="button" class="close" data-dismiss="modal">&times;</button>
<h4 class="modal-title">Request research list</h4>
```

If you want you could fix it here, otherwise we could make a new PR!

File: scout/server/blueprints/variants/templates/variants/str-variants.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 {% block content_main %}
 <form>
+  {{ variant_type }}
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/str-variants.html]: This was probably used for debugging, right?

File: scout/server/blueprints/variants/templates/variants/utils.html [hunk 2]
```diff
@@ -250,7 +251,7 @@ <h6>Phenotype terms (HPO):</h6>
   <div class="row justify-content-between" style="margin-top:20px;">
     <div class="col-6 text-left">
       <a href="{{ url_for('variants.variants', institute_id=institute._id, case_name=case.display_name,
-        variant_type='clinical', gene_panels=case.panels|selectattr('is_default')|map(attribute='panel_name')|list) }}"
+        variant_type=form.variant_type.data, gene_panels=case.panels|selectattr('is_default')|map(attribute='panel_name')|list) }}"
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/utils.html]: 👍

**Comment by northwestwitch** [comment]
> Hm, I guess - a little complicated to make (simple) automated tests for: suggestions are welcome.

It can be tested by capturing the HTML code of the pages. For instance look at the link when you hover on the reset button of the research SVs page (bottom left corner). Click on the figure or you won't see it:

image

**Comment by dnil** [comment]
> Also the "First page" button on the research SVs leads you to the clinical SV page (Should be fixed by the fix of point one I guess).

I could not repeat that, but it may have been precisely as you guessed, a residual from that "reset filter" jump out to clinical.

I've added the tests as suggested: much better than nothing - would have caught the last error at least! :+1:

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,14 +4,25 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## [4.8.3]
+## [x.x.x]
```
Comment (northwestwitch) [inline CHANGELOG.md]: Oh yes, I've tried to re-create this yesterday but I forgot the x.x.x

**Pull request merged**